### PR TITLE
Add 'pixart-rf-GCBBTM25.json' for device tests.

### PR DESCRIPTION
**Pull request by sam412081go**
Support GCBBTM25 BT mouse.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [*] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Review by superm1 — COMMENTED** [review]
Can you please add it to `data/device-tests/meson.build` too?

**Comment by sam412081go** [comment]
> Can you please add it to `data/device-tests/meson.build` too?

Added.

File: data/device-tests/pixart-rf-GCBBTM25.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "MOUSE-GCBBTM25",
+  "interactive": false,
+  "steps": [
+    {
+      "url": "https://www.fwupd.org/downloads/c5b40f0e4a425c7332513e099a4fd8600feb33e1893b590156fafdbbfe8f0929-GCBBTM25_v0001.cab",
```
Comment (superm1) [inline data/device-tests/pixart-rf-GCBBTM25.json:6]: This should be `emulation-url`, no?

File: data/device-tests/pixart-rf-GCBBTM25.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        {
+          "version": "0.0.1",
+          "guids": [
+            "657c927f-1889-596d-a0f1-d2abd9c067ab"
+          ]
+        }
+      ]
+    },
+    {
+      "url": "https://www.fwupd.org/downloads/b6bfa9c1f47252fb28d67bc3bf65bb9f11e2a3c1a32645450efdbdb64350c3c4-GCBBTM25_v0100.cab",
```
Comment (superm1) [inline data/device-tests/pixart-rf-GCBBTM25.json:17]: This should be `emulation-url`, no?

Comment (sam412081go) [reply data/device-tests/pixart-rf-GCBBTM25.json:17]: Due to some issues with the emulation data(https://github.com/fwupd/fwupd/pull/7330), it hasn't been uploaded yet.I will prepare the hardware to hughsie for debug.

So, there's only a link to the cab currently. Is the emulation URL necessary?

Comment (superm1) [reply data/device-tests/pixart-rf-GCBBTM25.json:17]: Oh I see. Then yes this is sensible for now and once emulation data is fixed that URL can add later.

Thanks!

**Review by superm1 — APPROVED** [review]

File: data/device-tests/pixart-rf-GCBBTM25.json (hunk 2 above)
Comment (sam412081go) [reply data/device-tests/pixart-rf-GCBBTM25.json:17]: Thank you!

**Pull request merged**